primitive.cabal

```diff
@@ -52,7 +52,7 @@ Library
 
   Build-Depends: base >= 4.5 && < 4.12
                , ghc-prim >= 0.2 && < 0.6
-               , transformers >= 0.2 && < 0.6
+               , transformers >= 0.4 && < 0.6
```

Comment:
Why are you bumping the upper version bounds for `transformers`? (Note that that is what is causing the GHC 7.8 build failure in Travis.)

Reply:
If we can keep the bounds down, maybe we can just deprecate the last couple versions altogether. The correctness and safety bugs are really piling up.